bash/ostree

```diff
@@ -180,542 +180,6 @@ _ostree_ostree() {
-            local argpos=$( __ostree_pos_first_nonflag $( __ostree_to_alternatives "$options_with_args" ) )
-
-            if [ $cword -eq $argpos ]; then
-                __ostree_compreply_refs
-            fi
-    esac
-
-    return 0
-}
-
-_ostree_admin_undeploy() {
-    local boolean_options="
-        $main_boolean_options
-    "
-
-    local options_with_args="
-        --sysroot
-    "
-
-    local options_with_args_glob=$( __ostree_to_extglob "$options_with_args" )
-
-    case "$prev" in
-        --sysroot)
-            __ostree_compreply_dirs_only
-            return 0
-            ;;
-        $options_with_args_glob )
-            return 0
-            ;;
-    esac
-
-    case "$cur" in
-        -*)
-            local all_options="$boolean_options $options_with_args"
-            __ostree_compreply_all_options
-            ;;
-    esac
-
-    return 0
-}
-
-_ostree_admin_unlock() {
-    local boolean_options="
-        $main_boolean_options
-        --hotfix
-    "
-
-    local options_with_args="
-        --sysroot
-    "
-
-    local options_with_args_glob=$( __ostree_to_extglob "$options_with_args" )
-
-    case "$prev" in
-        --sysroot)
-            __ostree_compreply_dirs_only
-            return 0
-            ;;
-        $options_with_args_glob )
-            return 0
-            ;;
-    esac
-
-    case "$cur" in
-        -*)
-            local all_options="$boolean_options $options_with_args"
-            __ostree_compreply_all_options
-            ;;
-    esac
-
-    return 0
-}
-
-_ostree_admin_upgrade() {
-    local boolean_options="
-        $main_boolean_options
-        --allow-downgrade
-        --deploy-only
-        --pull-only
-        --reboot -r
-    "
-
-    local options_with_args="
-        --os
-        --override-commit
-        --sysroot
-    "
-
-    local options_with_args_glob=$( __ostree_to_extglob "$options_with_args" )
-
-    case "$prev" in
-        --override-commit)
-            __ostree_compreply_commits
-            return 0
-            ;;
-        --sysroot)
-            __ostree_compreply_dirs_only
-            return 0
-            ;;
-        $options_with_args_glob )
-            return 0
-            ;;
-    esac
-
-    case "$cur" in
-        -*)
-            local all_options="$boolean_options $options_with_args"
-            __ostree_compreply_all_options
-            ;;
-    esac
-
-    return 0
-}
-
-_ostree_admin() {
-    local subcommands="
-        cleanup
-        config-diff
-        deploy
-        init-fs
-        instutil
-        os-init
-        set-origin
-        status
-        switch
-        undeploy
-        unlock
-        upgrade
-    "
-
-    __ostree_subcommands "$subcommands" && return 0
-
-    case "$cur" in
-        -*)
-            COMPREPLY=( $( compgen -W "$main_boolean_options" -- "$cur" ) )
-            ;;
-        *)
-            COMPREPLY=( $( compgen -W "$subcommands" -- "$cur" ) )
-            ;;
-    esac
-
-    return 0
-}
-
-_ostree_cat() {
```

Comment:
Looks like we're nixing `ostree cat` too here. :crying_cat_face:

Reply:
Hah, excellent use of the sad cat emoji! :smile:   Fixup :arrow_down: